.goreleaser.yml

```diff
@@ -5,47 +5,46 @@ release:
   github:
     owner: jpillora
     name: opts-cli
-  name_template: '{{.Tag}}'
+  name_template: "{{.Tag}}"
   # disable: true
 
 builds:
-- 
-  binary: opts-cli
-  env:
-  - CGO_ENABLED=0
-  goos:
-  - linux
-  - darwin
-  - windows
-  goarch:
-  - amd64
-  - "386"
-  ignore:
-  - goos: darwin
-    goarch: 386
-  main: .
-  ldflags:
-  - -s -w -X main.Version={{.Version}} -X main.Commit={{.Commit}} -X main.Date={{.Date}}
+  - binary: opts-cli
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+      - "386"
+    ignore:
+      - goos: darwin
+        goarch: 386
+    main: .
+    ldflags:
+      - -s -w -X main.version={{.version}} -X main.commit={{.commit}} -X main.date={{.date}}
```

Comment:
Ok, I was wrong you can stuff privates, but the templates need to by {{.Version}} etc.

goreleaser --skip-validate --skip-publish --rm-dist
```
   ⨯ release failed after 0.03s error=template: ldflags:1:24: executing "ldflags" at <.version>: can't evaluate field version in type struct { Commit string; Tag string; Version string; Date string; Env map[string]string }
```